internal/cmd/templatefuncs.go

```diff
@@ -482,6 +482,15 @@ func (c *Config) setValueAtPathTemplateFunc(path, value, dict any) any {
 	return result
 }
 
+func (c *Config) splitListTemplateFunc(sep, s string) []interface{} {
```

Comment:
nit: Please use `any` instead of `interface{}`.

Reply:
Should I also make this change in the commit message?

Reply:
Yes please :)